packages/@react-native-windows/codegen/src/Cli.ts

```diff
@@ -209,6 +220,8 @@ function generate(
   );
 
   const generateNM2 = createNM2Generator({namespace: argv.namespace});
+  const generateTypeScript = createTypeScriptGenerator();
```

Comment:
>const generateTypeScript [](http://example.com/codeflow?start=2&length=24)

Should we move it inside of the `if (argv.ts)` below to avoid creation of the generator if it is not needed?